app/src/main/java/de/westnordost/streetcomplete/osm/ResurveyUtils.kt

```diff
@@ -20,8 +20,8 @@ val LAST_CHECK_DATE_KEYS = listOf(
     "survey_date"
 )
 
-fun LocalDate.toCheckDateString(): String =
-    DateTimeFormatter.ISO_LOCAL_DATE.format(this)
+@Suppress("NOTHING_TO_INLINE")
+inline fun LocalDate.toCheckDateString(): String = this.toString()
```

Comment:
Please add a unit test that confirms that `toString` on the kotlinx.datetime.LocalDate actually outputs the date in the desired format

Reply:
https://github.com/streetcomplete/StreetComplete/pull/4471/commits/c31617ce3ae573285c9f06ea5c2c9758aa17a5c6

Reply:
Turns out there was already a test for this:
https://github.com/streetcomplete/StreetComplete/blob/0e01b1a92e4390806a100c2d28e24cd9a5dcc12e/app/src/test/java/de/westnordost/streetcomplete/data/meta/ResurveyUtilsTest.kt#L22-L24